presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -315,11 +314,6 @@ public PlanOptimizers(
                                 .addAll(new SimplifyCardinalityMap().rules())
                                 .addAll(new DesugarRowSubscript(metadata, sqlParser).rules())
                                 .build()),
-                new IterativeOptimizer(
```

Comment:
Why is this removed? Can you add detail in the commit?

Reply:
This logic is moved to TranslateExpressions